doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,300 @@
+==================================
+Gas-Solid Spouted Cylinder Bed
+==================================
+
+It is strongly recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_  and `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ for a detailed description on the concepts and physical meanings of the DEM and CFD-DEM parameters.
+This example is an extension of the `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_ for a cylindrical geometry. The boundary condition and some parameters have been changed accordingly.
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles`` and ``lethe-fluid-particles``
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+- Simulates a solid-gas cylinder-shaped spouted bed
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+Both files mentioned below are located in the example's folder (``examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed``).
+
+- Parameter file for CFD-DEM simulation of the spouted bed: ``gas-solid-spouted-bed.prm``
+- Parameter file for particle generation and packing: ``dem-packing-in-spouted-bed.prm``
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the spouting of spherical particles in air in a cylindrical geometry. As noted in the example of `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-beml>`_, we use ``lethe-particles`` to fill the bed with particles, and ``lethe-fluid-particles`` as a solver of CFD-DEM.
+
+-------------------
+DEM Parameter File
+-------------------
+
+Here, we will focus only on the parts that have been modified.
+
+Mesh
+~~~~~
+
+In this example, we are simulating a cylinder shaped spouted bed. We introduce the flow through a short channel that is connected to the inlet of the bed. A schematic image is shown below;
+
+.. image:: images/geometry.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry
+    :height: 10cm 
+
+The geometry of the bed was created using `Pointiwise <../../../tools/pointwise/pointowise.html>`_, and the overview of created mesh is also shown below;
+
+.. image:: images/mesh.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: mesh_ver
+    :height: 10cm
+
+In Unresolved-CFD-DEM, Cells need to have a enough volume to contain particles in it. Basically the size of cells should be least three time larger than the diameter of the particles in order to get stable calculation. So this can be expressed as follows.
```

Comment:
```suggestion
In Unresolved CFD-DEM, the averaging volume used to calculate the void fraction needs to be large enough to contain several particles (>10). Since the averaging volume used in the quadrature-centred method is generally related to the cell volume, this introduces a limitation on the cell size. In general, the averaging volume, which in this case corresponds to the cell size, should be approximatively three time larger than the diameter of the particles in order to get stable calculation. So this can be expressed as follows:
```